Documents/Technical Specifications/Technical.md

```diff
@@ -0,0 +1,494 @@
+
+# Gameplay
+
+## Objective
+
+The objective of the game is to reach the cup of coffee. The cup of coffee is the end of the game.
+The game will be separated in levels. Each level will move the cup of coffee further away from the player.
+
+## Camera
+
+## Mechanics
+
+Each mechanic will be explained in details here.
+
+### Movement
+
+The player will be able to move on 2 axis (X / Y by default).
+The player will be able to move at a constant speed.
+The player will be able to move in 4 directions (Up / Down / Left / Right).
+Each direction will be mapped to a key.
+
+- Forward: Z
+- Backward: S
+- Left: Q
+- Right: D
+
+Each key when pressed will trigger a movement in the corresponding direction.
+The will also trigger the associated animation for the cube to move.
+While the cube is moving, any other interractions will be disabled to avoid unwanted behavior.
+
+### Obstacle
+
+Obstacles will be placed in the level to prevent the player from reaching the cup of coffee.
+Obstacles will be items integrated in the environment and might take any forms ( Wall, plate, etc. ).
+Each obstacle will have a collision box for the player to not glitch through it.
+Each obstacle will have a material to make it visible.
+Each obstacles placement and details will be define in the [Level section](#level).
+
+### Sticky
+
+Sticky will be a mechanic that will allow the player to stick to walls.
+It will be shown as a green particle effect on the wall as if the wall was dirty.
+It will activate when the player enter a special zone.
+And will stay activated as long as the player stay in that zone and follow the predefined path.
+The controls stay relative to the camera.
+It means old controls become as follow:
+
+- Forward: Z -> Forward
+- Backward: S -> Backward
+- Left: Q -> Down
+- Right: D -> Up
+  
+If the player is sticked to a wall and decide to leave the predefined path, the sticky mechanic will be deactivated and the player will fall resulting in a death.
+
+### Catapult
+
+Catapult will be a mechanic that will allow the player to launch itself in the air.
+It will be represented as a spoon.
+It will activate when the player enter the back of the spoon.
+It will launch the player in the direction of the spoon from point A to point B.
+
+There is no laucnh control, air control or gravity control.
+The controls will be desactivated while the player is in the air.
+The catapult have an animation.
+
+### Teleport
+
+Teleport will be a mechanic that will allow the player to teleport to a specific location.
+It will be represented as mouse hole.
+It will activate when the player enter the mouse hole.
+It will teleport the player to the other side of the mouse hole.
+The teleport will be instant.
+It will be one sided ( No come back ).
```

Comment:
Are you sure about this statement ?

Reply:
For the moment, it will be easier for Game design, it might change in the future if we think it will add a big difference in the gameplay